src/main/java/com/stuypulse/robot/Constants.java

```diff
@@ -55,4 +81,81 @@ public interface Constraints {
 		double MAX_ANGULAR_VEL = (MAX_VEL * 2) / TRACK_WIDTH_METERS;
 		double MAX_ANGULAR_ACC = (Math.pow(MAX_ACC, 2)) / (TRACK_WIDTH_METERS/2); // a = v^2 / r
 	}
+
+	public interface EdwinSettings {
+        // If speed is below this, use quick turn
+        SmartNumber BASE_TURNING_SPEED = new SmartNumber("Driver Settings/Base Turn Speed", 0.4);
+
+        // Low Pass Filter and deadband for Driver Controls
+        SmartNumber SPEED_DEADBAND = new SmartNumber("Driver Settings/Speed Deadband", 0.05);
+        SmartNumber ANGLE_DEADBAND = new SmartNumber("Driver Settings/Turn Deadband", 0.05);
+
+        SmartNumber SPEED_POWER = new SmartNumber("Driver Settings/Speed Power", 1.0);
+        SmartNumber ANGLE_POWER = new SmartNumber("Driver Settings/Turn Power", 1.0);
+
+        SmartNumber SPEED_FILTER = new SmartNumber("Driver Settings/Speed Filtering", 0.2);
+        SmartNumber ANGLE_FILTER = new SmartNumber("Driver Settings/Turn Filtering", 0.01);
+
+        // Current Limit for the motors
+        int CURRENT_LIMIT_AMPS = 60;
+
+        // If the motors are inverted
+        boolean IS_INVERTED = true;
+
+        // Width of the robot
+        double TRACK_WIDTH = Units.inchesToMeters(30.0); // SEAN PROMISED !
+
+        boolean USING_GRAYHILLS = false;
+        boolean USING_GYRO = true;
```

Comment:
remove these.. ... ah i see what went wrong now. USING_GRAYHILLS in the Edwin branch was false because we used to have code that allowed us to do both